packages/office-ui-fabric-react/src/components/Coachmark/Coachmark.tsx

```diff
@@ -202,13 +221,36 @@ export class Coachmark extends BaseComponent<ICoachmarkTypes, ICoachmarkState> {
                     color={color}
                   />
                 )}
-                <FocusZone>
-                  <div className={classNames.entityHost} data-is-focusable={true} onFocus={this._onFocusHandler}>
-                    <div className={classNames.entityInnerHost} ref={this._entityInnerHostElement}>
+                <FocusTrapZone isClickableOutsideFocusTrap={true} forceFocusInsideTrap={false}>
```

Comment:
What's the point of having this focustrap zone if you can click out of it and it doesn't force focus within it?

Reply:
Removed the `forceFocusInsideTrap` prop.  I think I inadvertently added this and forgot to remove it.   I think we still want to have a clickable region outside of the trap though.